app/views/static/ems_cloud/arbitration_profile_edit.html.haml

```diff
@@ -0,0 +1,109 @@
+%form.form-horizontal#form_div{"name" => "angularForm"}
+  %div
+    %div
+      .form-group{"ng-class" => "{'has-error': angularForm.name.$invalid}"}
+  %div
+    %div
+      .form-group{"ng-class" => "{'has-error': angularForm.name.$invalid}"}
+        %label.col-md-2.control-label{"for" => "textInput-markup"}
```

Comment:
the value of `for` here should match the `id` (or `name`) of the input element

and same for the other ones, "textInput-markup" is a copypaste we see too often.. :)